runtime/compiler/runtime/SignalHandler.c

```diff
@@ -1777,105 +1779,109 @@ UDATA jitAMD64Handler(J9VMThread* vmThread, U_32 sigType, void *sigInfo)
 		} else {
 			void * stackMap;
 			UDATA registerMap;
-			switch (sigType) {
-			case J9PORT_SIG_FLAG_SIGFPE_DIV_BY_ZERO:
-			case J9PORT_SIG_FLAG_SIGFPE_INT_DIV_BY_ZERO:
+			BOOLEAN isCFGEnabled = J9_ARE_ANY_BITS_SET(vmThread->javaVM->sigFlags, J9_SIG_WINDOWS_MITIGATION_POLICY_CFG_ENABLED);
+
```

Comment:
While the logic here seems sound, there was a lot of offline discussion and exploration that led to this code change.  Can you introduce a comment in the code with a few sentences summarizing that and describing why we are making such a change in the JIT signal handler for Windows CFG?  This may be helpful down the road for someone not involved in those investigations to understand why these restrictions are put in place here.

Reply:
Addressed in [b4a08b0](https://github.com/eclipse-openj9/openj9/compare/5299640caa23f53964286a7a35e6e85052b01a96..b4a08b0f76e5daeb58f44cc05be09938b114a367). Ready for another review. Thanks!